dbld/pip_packages.manifest

```diff
@@ -8,6 +8,7 @@ hyrule                  [devshell, kira]                                       [
 requests                [kira]                                                 [python2, python3]
 
 # pip packages for python functional tests
+networkx                [centos, fedora, debian, ubuntu, devshell]             [python3]
```

Comment:
if we want to do this, I think this package should be moved out of contrib and probably into scripts.

If we did that, I would also try to run it from inside of the new build-time Python venv (e.g. #4193). This would mean that networkx should be added to dev-requirements.txt in the root and not here.

We might also consider dropping cleaning up/getting rid off pip_packages.manifest.

Reply:
Thanks for the note. 
I have rebased this PR on top of #4193 . From now this PR is depends on #4193 (how to install `networkx` package)
I have also replaced the commit where `networkx` has added. The new installation place is `dev-requirements.txt` file in the root.
After this new smoke testcases could run successfully for me.
If I am correct @alltilla has future plans with `config_option_database`. Could you please help a little for us?